<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>762</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>488</width>
    <height>762</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>488</width>
    <height>762</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>488</width>
      <height>762</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>488</width>
      <height>762</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>488</width>
      <height>762</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(28, 27, 32);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="down1">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>660</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Product Sans Medium</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgb(233, 231, 248);
border-radius: 25px;
color: rgb(53, 45, 101);
}
QPushButton:hover {
background-color: rgb(240, 238, 255);
}
QPushButton:pressed {
background-color: rgb(217, 215, 230);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="url_set">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>570</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>81</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>81</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Product Sans Medium</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgb(233, 231, 248);
border-radius: 25px;
color: rgb(53, 45, 101);
}
QPushButton:hover {
background-color: rgb(240, 238, 255);
}
QPushButton:pressed {
background-color: rgb(217, 215, 230);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="url_edit">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>400</y>
       <width>442</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Product Sans</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
background-color: rgb(52, 55, 74);
color: rgb(215, 215, 230);
border-radius: 35px;
padding-left: 20px;
padding-right: 20px;
}</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>insert your link</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>100</y>
       <width>442</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>ProductBold</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: qlineargradient(spread:pad, x1:0, y1:0.5, x2:1, y2:0.500136, stop:0 rgba(201, 191, 254, 255), stop:0.43 rgba(126, 120, 159, 255), stop:1 rgba(73, 62, 139, 255));
}</string>
     </property>
     <property name="text">
      <string>Hello, User</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>220</y>
       <width>442</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>ProductBold</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>video</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>160</y>
       <width>442</width>
       <height>70</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>442</width>
       <height>70</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>ProductBold</family>
       <pointsize>36</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Letâ€™s download</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
